src/scrabble.js

```diff
@@ -1,5 +1,44 @@
-function scrabble() {
+const {
+  stringIsValid,
+  findDoubleChars,
+  findTripleChars,
+  getScoreOfPlainString,
+  outermostBracketsAreDouble,
+  outermostBracketsAreTriple
+} = require('./scripts.js')
 
+function scrabble(string) {
```

Comment:
I will need more time to complete my review but it looks really amazing how you structure everything and. Also, your thought process is very clear.